xbmc/GUIInfoManager.cpp

```diff
@@ -1798,7 +1798,7 @@ const infomap system_labels[] = {{"hasnetwork", SYSTEM_ETHERNET_LINK_ACTIVE},
                                  {"batterylevel", SYSTEM_BATTERY_LEVEL},
                                  {"friendlyname", SYSTEM_FRIENDLY_NAME},
                                  {"alarmpos", SYSTEM_ALARM_POS},
-                                 {"isinhibit", SYSTEM_ISINHIBIT},
```

Comment:
Can you re-add this line as

```c++
{"isinhibit", SYSTEM_IDLE_SHUTDOWN_INHIBITED}, // Deprecated
```

That way, if skins or add-ons don't update, they won't break.

Reply:
@garbear : Updated